config/database.yml

```diff
@@ -21,7 +21,8 @@ default: &default
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   user: postgres
-  host: operationcode-psql
+  # host: operationcode-psql
```

Comment:
@rickr - thoughts on this change?

Reply:
@hpjaj I just made this change to get it working on my local machine. I'm not sure how I could have gotten it to run the server otherwise.